opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -123,14 +178,23 @@ def __init__(self, name, version=None, schema_url=None):
 
     @property
     def name(self):
+        """
```

Comment:
Shouldn't this documentation be in the `__init__` method?

Reply:
These appear separately in the documentation: https://opentelemetry-python.readthedocs.io/en/latest/api/metrics.html#opentelemetry._metrics.Meter.name

I'll add them to the init as well in a follow up PR, since I need to add docs for instruments as well